dlworkspace-deployment/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterNameDesired": {
+            "value": "GEN-UNIQUE"
+        },
+        "adminPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "dlwsAdmins": {
+            "value": "GEN-UNIQUE"
+        },
+        "devSourceIP": {
```

Comment:
remove all the parameters that have defaultValues in the template

Reply:
done